### PR TITLE
Crazy Blockchain Tutorial

### PR DESCRIPTION
**Pull request by anusha-ctrl**

### CODE REVIEW

File: precompile/crazy_with_power.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+	// Return the packed output and the remaining gas
+	return packedOutput, remainingGas, nil
+}
+
+// createCrazyWithPowerPrecompile returns a StatefulPrecompiledContract with getters and setters for the precompile.
+
+func createCrazyWithPowerPrecompile(precompileAddr common.Address) StatefulPrecompiledContract {
+	var functions []*statefulPrecompileFunction
+
+	methodBenevelontGod, ok := CrazyWithPowerABI.Methods["benevelontGod"]
```
Comment (darioush) [inline precompile/crazy_with_power.go:193]: seems like a typo: benevolent

Comment (anusha-ctrl) [reply precompile/crazy_with_power.go:193]: haha thanks

File: contract-examples/contracts/ICrazyWithPower.sol [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+interface ICrazyWithPower {
+  // wreakHavoc() chooses a random address to subtract funds from
+  function wreakHavoc() external;
+
+  // setGreeting chooses a random address to add funds to
```
Comment (darioush) [inline contract-examples/contracts/ICrazyWithPower.sol:12]: seems this comment should be updated

File: precompile/crazy_with_power.go [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+	CrazyWithPowerRawABI = "[{\"inputs\":[],\"name\":\"benevelontGod\",\"outputs\":[],\"stateMutability\":\"nonpayable\",\"type\":\"function\"},{\"inputs\":[],\"name\":\"wreakHavoc\",\"outputs\":[],\"stateMutability\":\"nonpayable\",\"type\":\"function\"}]"
+)
+
+// CUSTOM CODE STARTS HERE
+// Reference imports to suppress errors from unused imports. This code and any unnecessary imports can be removed.
+var (
+	_ = errors.New
+	_ = big.NewInt
+	_ = strings.NewReader
+	_ = fmt.Printf
```
Comment (darioush) [inline precompile/crazy_with_power.go:54]: should we remove these if not used?

File: precompile/crazy_with_power.go [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+	if err != nil {
+		panic(err)
+	}
+	CrazyWithPowerABI = parsed
+
+	CrazyWithPowerPrecompile = createCrazyWithPowerPrecompile(CrazyWithPowerAddress)
+}
+
+// NewCrazyWithPowerConfig returns a config for a network upgrade at [blockTimestamp] that enables
+// CrazyWithPower .
```
Comment (darioush) [inline precompile/crazy_with_power.go:83]:
```suggestion
// CrazyWithPower.
```

File: precompile/crazy_with_power.go [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+var (
+	_ StatefulPrecompileConfig = &CrazyWithPowerConfig{}
+
+	CrazyWithPowerABI abi.ABI // will be initialized by init function
+
+	CrazyWithPowerPrecompile StatefulPrecompiledContract // will be initialized by init function
+)
+
+// CrazyWithPowerConfig implements the StatefulPrecompileConfig
+// interface while adding in the CrazyWithPower specific precompile address.
```
Comment (darioush) [inline precompile/crazy_with_power.go:67]: not sure this comment makes sense? (adding in the CrazyWithPower specific precompile config? functionality?). maybe this should be updated in the generator?

**Pull request closed without merging**